src/components/Button/Button.tsx

```diff
@@ -0,0 +1,37 @@
+import './Button.scss';
+import classNames from 'classnames';
```

Comment:
You need to commit `package.json` file with the updated packages list. Now GitHub tests can't start.

Reply:
I have added file but tests are still failing.